.github/workflows/trigger_ci.yml

```diff
@@ -17,4 +17,8 @@ jobs:
       - name: Trigger Pipeline
         run: |
           #!/bin/bash
-          curl --fail --request POST --form token=${{ secrets.PIPELINE_TOKEN }} -F ref=${GITHUB_HEAD_REF}  "${{ secrets.PIPELINE_URL }}"
+          curl --fail --request POST \
+            --form token=${{ secrets.PIPELINE_TOKEN }} \
+            --form variables[TRITON_PERF_ANALYZER_REPO_TAG]=${GITHUB_HEAD_REF} \
+            --form variables[TRITON_CLIENT_REPO_TAG]=fpetrini-dummy \
```

Comment:
Isn't it a hardcoded value?

Reply:
`TRITON_CLIENT_REPO_TAG`? Yes, I envision this value being hardcoded to `main`. It would be the developer's responsibility to update this value when necessary and restore it prior to merging. This behavior is similar to the one we have adopted in gitlab.

Reply:
I would suggest to set default value to the default `main` branch and communicate the same with developers.

Reply:
Yes, that's what I stated above. This is just a draft PR where I was trying to show that branch name changes will take effect. I can switch it back now and open it as a true PR if you find this solution to the slack thread acceptable.